apps/moose-console/src/lib/snippets.ts

```diff
@@ -103,20 +103,15 @@ export const curlSnippet = (data: CliData, model: DataModel) => {
   const ingestionPoint = getIngestionPointFromModel(model, data);
   const columns = createColumnStubs(model);
 
-  return `
-  curl -X POST -H "Content-Type: application/json" -d '{${columns.join()}}' \\
-  http://${data.project && data.project.local_webserver_config.host}:${data.project.local_webserver_config.port}/${ingestionPoint.route_path}
-`;
+  return `curl -X POST -H "Content-Type: application/json" -d '{${columns.join()}}' \\
+    http://${data.project && data.project.local_webserver_config.host}:${data.project.local_webserver_config.port}/${ingestionPoint.route_path}`;
 };
 
 export const bashSnippet = (data: CliData, model: DataModel) => {
   const curlCommand = curlSnippet(data, model);
 
-  return `
-  #!/bin/bash
-
+  return `#!/bin/bash
```

Comment:
Looking like you have some indent issues here. Recommend newlining at the top and then making it easy to scan the snippets by sticking them all to the leftmost indent

Reply:
I fixed the indents and some of the snippets in my PR if you rebase

Reply:
New lining adds a new line to the displayed code block. 
<img width="706" alt="image" src="https://github.com/514-labs/moose/assets/4805522/0e7bbef9-57d5-4e66-b8af-542ac696d6a6">

Opted for adding a backslash. A lil weird but also you to scan 
<img width="180" alt="image" src="https://github.com/514-labs/moose/assets/4805522/d8481d4a-2475-4766-b003-99cea5ea786f">